tools/repo_diff/README.md

```diff
@@ -0,0 +1,20 @@
+# Import check
+
+1. Get database dumps and unzip them into `input/dump`, rename to `dspace6.sql` and `dspace7.sql` accordingly.
+ 
+1. Create `dspace6` and `dspace7` databases from dump:
```

Comment:
Step 2. not 1.

Reply:
these are automatically incremented
https://www.markdownguide.org/basic-syntax/#ordered-lists